packages/react-core/src/components/DescriptionList/DescriptionListDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+import { css } from '@patternfly/react-styles';
+
+export interface DescriptionListDescriptionProps extends React.HTMLProps<HTMLElement> {
+  /** Anything that can be rendered inside of list description */
+  children: React.ReactNode;
+  /** Additional classes added to the DescriptionListDescription */
+  className?: string;
+  /** Indicates which component will be used for the text wrapped by the description */
+  textComponent?: React.ReactNode;
+  /** Default hyperlink location */
+  href?: string;
+  /** Icon added to the left of the label text. */
+  icon?: React.ReactNode;
```

Comment:
You don't need this either.  The consumer will take care of adding the icon iff they choose.